opennms-doc/guide-admin/src/asciidoc/text/flows/setup.adoc

```diff
@@ -110,7 +110,7 @@ parsers.0.class-name org.opennms.netmgt.telemetry.protocols.netflow.parser.Netfl
 NOTE: The associated protocol, in this case `Netflow-5` must also be enabled on _{opennms-product-name}_ for the messages to be processed.
 
 In some scenarios the exporters address is altered due to network address translation.
-In this case you can use node meta-data to identify the exporter.
+In this case you can use node data to identify the exporter.
```

Comment:
It's the node's metadata that is used to identify the exporter.

Reply:
Typo. Thanks.